main.go

```diff
@@ -114,16 +117,18 @@ func main() {
 			if isProjection {
 				secondMonthHeader += " (projection)"
 			}
-			tw.AppendHeader(table.Row{"Service", firstMonthStart, secondMonthHeader, "Delta"})
+			tw.AppendHeader(table.Row{"Service", firstMonthStart, secondMonthHeader, "Delta", "Delta Percent"})
 			for service, amounts := range finalResultsCosts {
-				tw.AppendRow(table.Row{service, ac.FormatMoney(amounts[0]), ac.FormatMoney(amounts[1]), ac.FormatMoney(amounts[2])})
+				tw.AppendRow(table.Row{service, ac.FormatMoney(amounts[0]), ac.FormatMoney(amounts[1]), ac.FormatMoney(amounts[2]), fmt.Sprintf("%s%%%%", accounting.FormatNumber(amounts[3], 1, "", "."))})
```

Comment:
Not sure why I need to escape it like this `%%%%` to get a single `%` in the output. According to `fmt` docs it should only require `%%`. Suspect it might be some interaction with the `tw` code.